### PR TITLE
Explain Scene Tree Double-Click Feature

### PR DESCRIPTION
**Pull request by tn12787**
ditto.

### CODE REVIEW

**Comment by tn12787** [comment]
👍

File: guide/the-scene-tree.md [hunk 1]
```diff
@@ -20,9 +20,11 @@ of the VRML97 nodes and Webots nodes.
...
 
 Nodes can be expanded with a double-click. When a field is selected, its value
-can be edited at the bottom of the Scene Tree. All changes will be immediately
-reflected in the 3D window. The following buttons are available in the field
-editor section:
+can be edited at the bottom of the Scene Tree. Double-clicking or pressing the
+Enter key on a field selects the first editable item of the field editor panel.
+Keyboard focus can be returned to the Scene Tree by tabbing through all of the
+items in the field editor panel. All changes will be immediately reflected in
+the 3D window. The following buttons are available in the field editor section:
```
Comment (DavidMansolino) [inline guide/the-scene-tree.md]: I would also add that in order to toggle a boolean field the space bar should be used.

**Comment by tn12787** [comment]
I explained everything clearly for all field types. This should be perfectly clear now 😄

**Comment by tn12787** [comment]
👍

**Pull request merged**